pkgs/by-name/pk/pkl/set_commit_id.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/buildSrc/src/main/kotlin/BuildInfo.kt b/buildSrc/src/main/kotlin/BuildInfo.kt
+index 0264375..dfabed1 100644
+--- a/buildSrc/src/main/kotlin/BuildInfo.kt
++++ b/buildSrc/src/main/kotlin/BuildInfo.kt
+@@ -87,14 +87,7 @@ open class BuildInfo(project: Project) {
+   val commitId: String by lazy {
+     // only run command once per build invocation
+     if (project === project.rootProject) {
+-      val process = ProcessBuilder()
+-        .command("git", "rev-parse", "--short", "HEAD")
+-        .directory(project.rootDir)
+-        .start()
+-      process.waitFor().also { exitCode ->
+-        if (exitCode == -1) throw RuntimeException(process.errorStream.reader().readText())
+-      }
+-      process.inputStream.reader().readText().trim()
++      "@commit_id@"
```

Comment:
You can get rid of this and replace it with `-DcommitId=<commitId>` once https://github.com/apple/pkl/pull/954 is merged, and when building Pkl 0.28.